Sources/MyStyles/FontsRegistrar.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  File.swift
+//  
+//
+//  Created by Ciarán Mulholland on 12/10/2021.
+//
```

Comment:
Very silly thing (especially for a POC), but we don't need this header "stuff" do we?

Reply:
nah we don't, I've just been lazy lol